### DISCUSSION TITLE
解决`\icarus`文件夹下没有配置文件的问题

### DISCUSSION THREAD

**Discussion by have19713** [ppoffice/hexo-theme-icarus#1103] [Show and tell]
当`\icarus`中没有配置文件时，
只需要将hexo文件夹下的`_config.icarus.yml`改名成`_config.yml`复制粘贴至`\icarus`就好了

**Comment by ppoffice** [comment]
https://hexo.io/zh-cn/docs/configuration.html#%E4%BD%BF%E7%94%A8%E4%BB%A3%E6%9B%BF%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6
